php_memcached_private.h

```diff
@@ -48,7 +48,11 @@
 #include <ext/standard/basic_functions.h>
 
 #ifdef PHP_WIN32
+#if PHP_VERSION_ID >= 80000
+#include "php_stdint.h"
```

Comment:
Please indent two spaces and add a second space to the alternate win32 include line

Reply:
> I don't have a win32 test machine myself. If this works for you I'll take your word for it. Is there a release note that we can link to in the comment indicating this was a known change in PHP 8.x?

i compiled dll file
basic usage works fine